src/v/redpanda/admin/api-doc/debug.json

```diff
@@ -1217,4 +1230,23 @@
                 }
             }
         },
+        "broker_id_override": {
+            "id": "broker_id_override",
+            "description": "An override for broker id and UUID",
+            "properties": {
+                "current_node_uuid": {
+                    "type": "string",
+                    "description": "Current node UUID, used to validate if request is addressed to the correct node"
+                },
+                "new_node_id": {
+                    "type": "int",
+                    "description": "New node id for current node"
```

Comment:
this is not needed, right?  the node will be assigned its old node_id anyway during booststrap even if it is is empty.

Reply:
it is needed in case there is no majority in the cluster i.e. node will not be able to join